index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <h1 id="title">Tell Me About Music</h1>
```

Comment:
I think it would be better to use class attribute than id attribute

Reply:
- [ ] Great job, identifying that there is a semantic issue here. It would be nice to explain why making the change to a `id` attribute is recommended. Cheers 👍

Reply:
Because we can repeat the class attribute in different groups and use it with the same names in different groups and tags. But the general feature of id is only for an element or a tag group. Therefore, here it is possible that we use different h1 on our page, so we need to give them the same style and this will increase the readability of our code.

Reply:
- [OPTIONAL] Kindly, merge both comments 💯 

> For example

I think it would be better to use `class` attribute than `id` attribute👌Because we can repeat the class attribute in different groups and use it with the same names in different groups and tags. But the general feature of id is only for an element or a tag group. Therefore, here it is possible that we use different h1 on our page, so we need to give them the same style and this will increase the readability of our code. 💯